src/relayclient/ContractInteractor.ts

```diff
@@ -256,8 +257,9 @@ export default class ContractInteractor {
     return this.web3.eth.getGasPrice()
   }
 
-  async getTransactionCount (address: string, defulatBlock?: BlockNumber): Promise<number> {
-    return this.web3.eth.getTransactionCount(address)
+  async getTransactionCount (address: string, defaultBlock?: BlockNumber): Promise<number> {
+    // @ts-ignore (web3 does not define 'defaultBlock' as optional)
+    return this.web3.eth.getTransactionCount(address, defaultBlock)
```

Comment:
nitpic: I'd prefer doing `defaultBlock as BlockNumber` instead of "ts-ignore" - since "ts-ignore" ignores everything, not only one error type

Reply:
I dream of a day when I will go over our ts-ignores and open PRs to relevant repos to fix their typings. This day is near, I just need to keep faith.

Reply:
no, I meant how to write a code with cast instead of ts-ignore. but its a nitpic - better writing int he future..

Reply:
ok, I will. But ```var_A: Something | undefined as Something``` is equivalent to ```var_A!```, which we forbid in linter for being not good.